mkosi.images/base/mkosi.build.chroot

```diff
@@ -70,7 +70,7 @@ if [ ! -f "$BUILDDIR"/build.ninja ]; then
         -D tests=unsafe
         -D slow-tests="${SLOW_TESTS:-false}"
         -D create-log-dirs=false
-        -D pamconfdir=no
+        -D pamconfdir=/usr/lib/pam.d/
```

Comment:
Can we get some justification for this change? I disabled this because our own snippets conflict with the ones installed by the distribution.

Reply:
we install a new pam snippet, which is used for uid0 sessions. Without this we'd not have it and one cannot use the tool